### PR TITLE
* added missing closing bracket

### PR DESCRIPTION
**Pull request by lahmann**
...strong competitor to vufind-org@824b8a94 comparing ascii chars ;)

### CODE REVIEW

File: themes/bootstrap3/js/vudl/record.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ function ajaxLoadPages(min, max) {
   }
```
Comment (crhallberg) [inline themes/bootstrap3/js/vudl/record.js]: I think it should go at the end of this line to close the `.done(`

**Comment by lahmann** [comment]
You are right - it's probably more efficient to close this pull request and let you fix the missing bracket in vufind-org/master.

**Comment by crhallberg** [comment]
Fixing now, thank you!

**Comment by lahmann** [comment]
Great! Thanks!

**Pull request closed without merging**